libs/dh/metering-point/feature-overview/src/lib/dh-metering-point.component.ts

```diff
@@ -142,9 +144,14 @@ export class DhMeteringPointComponent {
   isEnergySupplierResponsible = computed(() => this.meteringPointDetails()?.isEnergySupplier);
 
   breadcrumbLabel = translateSignal('meteringPoint.breadcrumb');
+  onNavigationEnd_EffectTrigger = toSignal(
```

Comment:
Suggestion: if its possible, maybe use the navigation ended signal from the breadcrumb service?

Reply:
Looks like it's possible 👍